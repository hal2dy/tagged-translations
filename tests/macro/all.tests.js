runTests(__dirname, [], ['babel-macros'])
