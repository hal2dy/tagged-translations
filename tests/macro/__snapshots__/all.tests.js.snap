// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test macro.js 1`] = `
"

const name = 'Vinh Le';

\`Xin chào \${name} 👻\`;

\`Đặt hàng\`;

\`This text doesn't have any translation\`;

\`Tìm thấy \${itemFound} sản phẩm!\`;

\`Vui lòng chuyển khoản \${grandTotal} đến tài khoản này trong vòng \${time} ngày để hoàn tất đơn hàng\`;"
`;
